src/core/worker.js

```diff
@@ -411,6 +411,8 @@ class WorkerMessageHandler {
         ignoreErrors: data.ignoreErrors,
         isEvalSupported: data.isEvalSupported,
         fontExtraProperties: data.fontExtraProperties,
+        standardFontDataUrl: data.standardFontDataUrl,
```

Comment:
When you're adding options here, please also remember to provide default values in https://github.com/mozilla/pdf.js/blob/239d0097fa5c717ba8dde56e175ae006c9053df1/src/core/evaluator.js#L91-L97